### PR TITLE
feat(action): add option to ignore missing blob files

### LINKED ISSUE #184: Allow `ifdtool_entries` to skip on missing file
Add option to skip item in `ifdtool_entries` if the file is missing.

### PR DESCRIPTION
**Pull request by AtomicFS**
fixes #184 

It can be handy sometimes to ignore if a blob is defined, but the actual file is missing. For example for a matrix builds where one variation needs the blob and another one does not.

### CODE REVIEW

File: action/recipes/stitching_test.go [hunk 1]
```diff
@@ -141,6 +141,12 @@ type makeFile struct {
 }
 
 func (base makeFile) MakeMe() error {
+	log.Printf("Path:       %s", base.Path)
+	log.Printf("Content:    %s", base.Content)
+	log.Printf("SourcePath: %s", base.SourcePath)
+	pwd, _ := os.Getwd()
```
Comment (MDr164) [inline action/recipes/stitching_test.go:147]: I'm surprised the linter didn't complain here about the ignored error. It's totally fine here to drop it but still odd that you didn't need `nolint:errcheck`

File: action/recipes/stitching_test.go [hunk 2]
```diff
@@ -141,6 +141,12 @@ type makeFile struct {
 }
 
 func (base makeFile) MakeMe() error {
+	log.Printf("Path:       %s", base.Path)
```
Comment (MDr164) [inline action/recipes/stitching_test.go:144]: I'm not fully in the loop in regards of logging. Did we switch to slog or is log still the way to go?

Comment (AtomicFS) [reply action/recipes/stitching_test.go:144]: Normally we use `slog`, but in tests we use `log`.

File: action/recipes/stitching_test.go (hunk 1 above)
Comment (AtomicFS) [reply action/recipes/stitching_test.go]: True, not sure why that is, so I added `nolint:errcheck` to future-proof this.

File: action/recipes/stitching_test.go (hunk 2 above)
Comment (MDr164) [reply action/recipes/stitching_test.go:144]: Totally overlooked that this is a test file. It makes no big difference but maybe we should use `t.Log()` but that would require this function to also take `*testing.T` as an input parameter and might be out of scope for this PR.

Comment (AtomicFS) [reply action/recipes/stitching_test.go:144]: `t.Log()` might be better than `slog`. I will put this into my todo list with low priority.

**Review by MDr164 — APPROVED** [review]

**Pull request merged**